docs/chapters/workflow/git-workflow.rst

```diff
@@ -65,3 +65,26 @@ force-pushed to the forked repo, or they can be added on top of the existing com
 request will be updated either way.
 
 Whenever the team is satisfied with the changes, pull the changes in by accepting the pull request.
+
+Good commits
+============
+
+A good commit (or patch):
+
+#. is atomic: Each commit should contain only one logical change. Use of vague verbs (e.g "update")
+   and phrases like "this patch also" are usually a good indicator that commit is not atomic.
+
+#. has a good commit message associated with it.
```

Comment:
The rationale below gives a good (even though vague) guideline on what a good commit message would be. For more specific example, readers can follow the link provided.